tembo-cli/src/cmd/apply.rs

```diff
@@ -528,6 +574,32 @@ fn get_extensions(
     vec_extensions
 }
 
+fn get_extensions_controller(
```

Comment:
Now that we have `tembo-stacks` more types can be reused but will require significant work in CLI so need to convert it to use it with `merge_app_reqs`.